### DISCUSSION TITLE
Binance Websockets @aggTrade

### DISCUSSION THREAD

**Discussion by AD4910** [community/community#135938] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hallo zusammen,

ich möchte die @aggTrade mit Hilfe von Websockets für den 15 Minuten Zeitintervall überwachen. In den 15 Minuten sollen die Käufe und/oder Verkäufe akkumuliert werden. Nach den 15 Minuten soll eine automatische bereinigung statt finden und die Berechnung fängt von vorne an.
Mein Problem ist es: wenn die WebsocketVerbindung unterbrochen wird (Timeout, sent 1011 (internal error) keepalive ping timeout; no close frame received), werden die akkumilierten Daten auch verloren gehen und somit habe ich keinen aktuellen Stand.

Hat jemand von Euch eine Lösung für das Problem?

**Comment by ebndev** [comment]
Hi @AD4910, thanks for participating!

Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting to ensure discussions are adhering to our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md). 

We’ll be locking this post for now, but please feel encouraged to create a new discussion in English if you still need support.
